scene/main/window.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "window.h"
+#include "core/object/class_db.h"
```

Comment:
```suggestion
```
Unnecessary, please make sure you're not using any feature in your editor to automatically add includes, or make sure they are valid